### ISSUE TITLE
Error using Set-SPOCustomFontCatalog

### ISSUE THREAD

**Issue by despingu** [MicrosoftDocs/OfficeDocs-SharePoint#3498]
I'm opening this as a similar issue is already closed: https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/issues/3291
Today I tried to update our recently setup font catalog but encountered multiple errors in doing so.

First, I was using version 16.0.22921.12000 of the Microsoft.Online.SharePoint.PowerShell module. With this I got an additional authentication prompt as described in the documentation. Unfortunately, I wasn't able to confirm this prompt successfully as I'm always getting the following error:
```
Set-SPOCustomFontCatalog : The sign-in name or password does not match one in the Microsoft account system.
In Zeile:1 Zeichen:1
+ Set-SPOCustomFontCatalog -FontFolder "C:\Temp\Fonts\" -LibraryUrl "ht ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Set-SPOCustomFontCatalog], IdcrlException
    + FullyQualifiedErrorId : Microsoft.SharePoint.Client.IdcrlException,Microsoft.Online.SharePoint.PowerShell.SetSPO
   CustomFontCatalog
```

Because it was mentioned in the other issue that version 16.0.22413.12000 of the module could be used successfully, I tried it also. There another error is thrown:
```
Set-SPOCustomFontCatalog : The request message is too big. The server does not allow messages larger than 2097152
bytes.
In Zeile:1 Zeichen:1
+ Set-SPOCustomFontCatalog -FontFolder "C:\Temp\Fonts\" -LibraryUrl "ht ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Set-SPOCustomFontCatalog], ServerException
    + FullyQualifiedErrorId : Microsoft.SharePoint.Client.ServerException,Microsoft.Online.SharePoint.PowerShell.SetSP
   OCustomFontCatalog
```

**Comment by scanum** [comment]
Thank you for submitting feedback.
I think the best way forward is if you open a service ticket in your tenant so this can get resolved ASAP. Based on the outcome let me know if it is something that can be called out in the docs.
Please follow this link to contact support for business products:https://docs.microsoft.com/office365/admin/contact-support-for-business-products
Please keep us posted here on the resolution so we can feed whatever information you discover into the content.

**Comment by despingu** [comment]
Thanks for the response, in order to prepare a ticket today, I saw that a new version of the module was released.
After the update the second auth prompt isn't using basic authentication anymore, so it's properly working.
Tested with module version 16.0.23019.12000. 

Only thing to mention: Out org asset library is called "Font Library" which could result in an url like this
"https://[tenant].sharepoint.com/sites/OrgAssets/Font**%20**Library"
When using this URL, I got the following error:

Set-SPOCustomFontCatalog : The library is not a valid font organization asset library

Using a space character instead of %20 worked totally fine however:
"https://[tenant].sharepoint.com/sites/OrgAssets/Font Library"
